web/src/scenes/team/MemberList/index.tsx

```diff
@@ -67,21 +59,22 @@ export const MemberList = memo(function MemberList() {
         </div>
 
         <div className="grid gap-y-4">
-          {members.map((member, key) => (
+          {filteredMembers.map((member, key) => (
             <div
               key={key}
               className="flex items-center bg-ffffff rounded-xl shadow-lg p-4 gap-3"
             >
               <div className="relative w-10 h-10 grid place-items-center bg-success-light rounded-full">
                 <Icon name="user-solid" className="w-4 h-4 bg-success" />
 
-                {member.verified && (
-                  <Icon
-                    name="badge-verification"
-                    className="absolute w-5 h-5 -bottom-1 -right-1"
-                    noMask
-                  />
-                )}
+                {/* FIXME: add verified flag to hasura */}
```

Comment:
@paolodamico we have is_subscribed field in user table. Is this field defines is user verified or not?